.circleci/config.yml

```diff
@@ -588,11 +588,11 @@ jobs:
           name: deploy to preview
           working_directory: ~/project/bin/preview-deploy
           command: |
-            if [ -n "$CI_PULL_REQUESTS" ]; then
+            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
               apt-get -qq update
               apt-get -qq install jq -y
               pip install --quiet awscli
-              PRNUM=$(basename $CI_PULL_REQUESTS)
+              PRNUM=$(basename $CIRCLE_PULL_REQUEST)
```

Comment:
`CI_PULL_REQUESTS` has been deprecated in favor of `CIRCLE_PULL_REQUEST` https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables